lib/mongoid/errors/invalid_discriminator_key_target.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+module Mongoid
+  module Errors
+
+    # This error is raised when trying to set or get the 
+    # discriminator key on a child class.
+    class InvalidDiscriminatorKeyTarget < MongoidError
+      def initialize(class_name, superclass)
```

Comment:
Could you write an api doc comment for this method?

Reply:
yup!